src/main/java/name/neuhalfen/projects/crypto/bouncycastle/openpgp/keys/callbacks/Rfc4880KeySelectionStrategy.java

```diff
@@ -179,10 +191,10 @@ protected boolean isExpired(PGPPublicKey pubKey) {
     final boolean isExpired;
 
     if (hasExpiryDate) {
-      final Instant expiryDate = pubKey.getCreationTime().toInstant()
-          .plusSeconds(pubKey.getValidSeconds());
+      final Date expiryDate = new Date(pubKey.getCreationTime().getTime()
```

Comment:
This needs a comment, or better yet: Using the (ugly) Calendar API:

https://stackoverflow.com/questions/8035125/java-how-to-add-seconds-to-timestamp